caom2/caom2/diff.py

```diff
@@ -249,10 +249,12 @@ def _get_list_differences(expected, actual, parent):
                 if temp_report is None:
                     match_found = True
                     start_index = act_index + 1
-                    actual_index = actual_copy.index(a)
-                    actual_copy.pop(actual_index)
-                    expected_index = expected_copy.index(e)
-                    expected_copy.pop(expected_index)
+                    if a in actual_copy:
```

Comment:
I don't quite understand why this was needed. `a` is member in `actual` and `actual_copy` should contain it too, no?

Reply:
There's a later `.pop() `call on `actual_copy`, so `a` can only be expected to be present the first time through the for loop.